### PR TITLE
Added overpass diagnostics, and fixed DT issue w/ H2O_TEND from GMI

### PR DESCRIPTION
**Pull request by mmanyin**
Zero diff for PCHEM. Added changes that went into the CCM benchmark **bench_10-14-1_gmi_free_c180_72lev**,
primarily OVP diagnostics and a small bug in a GMI diagnostic (H2O_TEND).   Edits only made in GMI and ChemEnv files.

### CODE REVIEW

File: GEOS_ChemEnvGridComp.F90 [hunk 1]
```diff
@@ -452,6 +506,47 @@ subroutine Run2 ( GC, IMPORT, EXPORT, CLOCK, RC )
        if (associated(pr_snow))  deallocate(pr_snow)
     end if
 
+!!!! >>>>>>>>>>>>>>>>  OVP
+
+    IF ( ovp_setup_done == .FALSE. ) THEN
```
Comment (mathomp4) [inline GEOS_ChemEnvGridComp.F90:511]: @mmanyin This needs to be:
```fortran
IF ( ovp_setup_done .eqv. .FALSE. ) THEN
```
The standard says you can't use `==` for comparing logicals.

Comment (mathomp4) [reply GEOS_ChemEnvGridComp.F90:511]: I say this as GNU is failing because of it (Intel doesn't seem to care 😄 )

Comment (mmanyin) [reply GEOS_ChemEnvGridComp.F90:511]: Thanks for the reminder!  I will update the branch.

**Review by gmao-esherman — APPROVED** [review]
Tested and verified that this PR is zero-diff.

**Pull request merged**